crates/ironrdp-web/Cargo.toml

```diff
@@ -65,4 +65,4 @@ smallvec = "1.11"
 x509-cert = { version = "0.2", default-features = false, features = ["std"] }
 tap = "1"
 semver = "1"
-url = "2"
+reqwest = "0.11.22"
```

Comment:
Stick to `gloo_net` instead of `reqwest`. `reqwest` is much bigger than `gloo_net` (which don’t do much more than wrapping the browser API), and we don’t need all that. (WASM files should be as small as possible.)

Reply:
updated